core-bundle/src/Resources/contao/library/Contao/Model.php

```diff
@@ -1193,30 +1172,12 @@ public static function countAll()
 	 */
 	public static function getClassFromTable($strTable)
 	{
-		if (isset(static::$arrClassNames[$strTable]))
-		{
-			return static::$arrClassNames[$strTable];
-		}
-
-		if (isset($GLOBALS['TL_MODELS'][$strTable]))
+		if (null === ($strClass = $GLOBALS['TL_MODELS'][$strTable] ?? null))
```

Comment:
Why so complicated?

```php
if (!isset($GLOBALS['TL_MODELS'][$strTable])) {
    throw new …
}

return $GLOBALS['TL_MODELS'][$strTable];
```

Reply:
I prefer DRY for array access. But this is otherwise CS for me.

Reply:
@leofeyer Feel free to change if you like.